source/crud.txt

```diff
@@ -0,0 +1,94 @@
+=====================
+CRUD Operations Guide
+=====================
+
+.. default-domain:: mongodb
+
+.. toctree::
+   :caption: CRUD Operations
+   :hidden:
+
+   /crud/read-operations
+   /crud/write-operations
+   /crud/query-document
+   /crud/returned-data
+   /crud/compound-operations
+
+Overview
+--------
+
+MongoDB helps you store data persistently across different instances of
+your applications. CRUD (Create, Read, Update, Delete) operations allow
+you to work with the data stored inside of MongoDB. CRUD operations fall
+into two categories: read operations and write operations. All operation
+methods can be configured with a special ``options`` parameter.
+
+Read Operations
+---------------
+
+CRUD read operations help you get data out of MongoDB.
+Read operations provide flexibility so you can use them to
+view data in many different ways. While some read operations focus on
+allowing you to view data that exists in the database, other operations
+let you monitor the database for new data that matches certain criteria,
+and still other read operations let you apply a series of
+transformations and filters to data before returning a resulting
+emphemeral collection.
```

Comment:
- Agreed on your first point. Reworded as per your suggestion.

- While I see what you're saying for the second point (and even recognize that your suggestion is technically more correct vocabulary), I think the verbiage you suggest is a little intimidating/confusing for new users. I lean more toward my initial wording since it actually conveys the thought more effectively. Happy to discuss better terms though if you still feel like this could use improvement!

- agreed, that's sort of a jargon-y term anyway. Reworded as per your suggestion.